### ISSUE TITLE
NIMA: Neural Image Assessment

### ISSUE THREAD

**Issue by ghuijo** [yskim0/One-Day-One-Paper#68]
#### 논문 읽다가 추가로 찾아본 내용 정리
* IQA: 기술적 왜곡이 첨가된 이미지의 품질을 평가하는 것
* IAQA: 이미지의 아름다운 정도를 평가하는 것
* IQA와 IAQA 모두 사람의 주관적인 평가를 예측하는 것으로 객관적으로 확실한 라벨은 존재할 수 없음
→ 보통 여러 사람의 의견을 평균낸 것을 ground truth 라벨로 삼음
→ 그것을 보통 MOS or DMOR라고 부름
→ MOS or DMOS와 최대한 비슷한 출력값을 내는 모델을 만드는 것이 IQA와 IAQA의 실질적인 목표
* MOS: mean opinion score
→ 모집된 여러 명의 피실험자들이 점수를 매긴 것을 평균낸 것
* DMOS: differential mean opinion score
→ 어떤 왜곡된 이미지의 원본을 보고 점수를 매긴 것에서 그 왜곡된 이미지를 보고 점수를 매긴 것을 빼줌
→ 각 피실험자가 여러 장의 이미지에 매긴 점수들(DOS)을 Z-score로 표준화해줌.
→ 여러 피실험자들이 각 이미지에 부여한 Z-score들을 평균냄

## Abstract
* 현존하는 대부분의 방법들은 mean opinion score 만을 predict함.
* 본 논문에서는 human opinion scores의 distribution을 CNN을 써서 예측함.
* 모델 아키텍쳐가 복잡하지 않고, human perception과 correlation이 높아 신뢰할 수 있음.
* <No-reference quality assessment에 해당.>

## Introduction
* IQA에서 사용될 수 있는 모델과 IAQA 과제에 사용될 수 있는 모델을 제안.
* 이미지 분류 데이터셋에서 train된 모델을 다른 과제에 사용하는 transfer learning을 이용함.
* fine-tuning을 해줄 때 IQA 데이터셋 이용 / IAQA 데이터셋 이용의 차이
* 모델을 train시킬 때 주관 평가 점수들의 평균값(MOS or DMOS)이 아니라 주관 점수의 분포를 이용해서 모델을 훈련시킴.
* 품질 점수를 하나의 scalar 값으로 매기지 않고 예측된 점수들의 분포로 나타냄.
* IQA 과제를 위해서 pretrained VGG16 모델 사용.
* IAQA 과제를 위해서 pretrained Inception-v2 모델 사용.
* IQA용 모델의 fine-tuning에 TID2013 데이터셋 사용.
* IAQA용 모델의 fine-tuning에 AVA(Aesthetic Visual Analysis) 데이터셋 사용.
* Loss function로 squared EMD(Earth Mover's Distance) 사용.

## Proposed Method
<image src="https://user-images.githubusercontent.com/67627471/129480417-69e559f3-494d-4ddc-9ed1-37a8c1dd0d14.png" width="50%" height="50%" />

* image classifier 아키텍쳐 기반. (VGG16, Inception-v2, MobileNet을 모두 다뤄봄.)
* baseline CNN의 마지막 layer를 fc layer로 바꿈. 그 다음 soft-max activation이 오도록 함.
* baseline CNN의 가중치는 ImageNet dataset으로 학습시켜서 초기화하고, quality assessment에 대해 end-to-end 학습을 함.
* 학습 때, input images는 256 × 256로 조정하고, 224 × 224 크기의 crop은 랜덤하게 추출함.
→ 이렇게 하면 small datasets으로 학습할 때 잠재적인 오버피팅 문제를 줄여줌.
* 주어진 이미지에 대한 human ratings의  Ground truth distribution
→ p = [ps1, . . . , psN ] with s1 ≤ si ≤ sN
→ AVA랑 TID2013 datasets에서 N = 10이고, AVA에서 s1 = 1 & sN = 10이고, TID에서 s1 = 0 & sN = 9임.
* classification tasks에서 training loss로 Soft-max cross-entropy 사용.

## Experimental Results
* aesthetics quality assessment를 위한 모델 / technical quality assessment를 위한 모델 둘로 분리해 train시켰음.
* accuracy는 두 classes를 classification한 것으로 평가.
<image src="https://user-images.githubusercontent.com/67627471/129480471-60c353b1-0dd0-4b6e-bfbb-5060d58a7f4e.png" width="70%" height="70%" />

* AVA test set에 적용한 NIMA(Inception-v2) 결과.
<image src="https://user-images.githubusercontent.com/67627471/129480547-5fe32953-80a3-41c3-b2e8-30a835cec47e.png" width="80%" height="80%" />

* landscape라고 label 붙은 AVA 이미지들 중 일부. NIMA(VGG16)이 predict한 score & 괄호 안에 ground truth score.
<image src="https://user-images.githubusercontent.com/67627471/129480602-a64385b9-0671-42a6-94c9-d4d94248089c.png" />

##
* Inception-v2 모델 공부하기.
